<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureMatchingClass</class>
 <widget class="QMainWindow" name="FeatureMatchingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FeatureMatching</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="PUSHBUTTON_IMAGEDIRECTION">
          <property name="text">
           <string>ImageDirection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LINEEDIT_IMAGEDIRECTION"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="PUSHBUTTON_RESULTDIRECTION">
          <property name="text">
           <string>ResultlDirection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LINEEDIT_RESULTDIRECTION"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
        <item>
         <widget class="QPushButton" name="PUSHBUTTON_START">
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FeatureMatching.qrc"/>
 </resources>
 <connections/>
</ui>
